### DISCUSSION TITLE
Adding elements to scene after scene has been loaded via dict.

### DISCUSSION THREAD

**Discussion by foshyjoshy** [mitsuba-renderer/mitsuba3#1084] [Q&A] [answered]
Hi, I'm curious if it's possible to add items to a scene after it has been initially created with scene = mi.load_dict(scene_dict). Specifically, I'm looking to programmatically add lights to the scene that already contains my sensor and meshes. Thanks.

**Comment by njroussel** [comment, marked as answer]
Hi @foshyjoshy 

Here's maybe a more complete answer: https://github.com/mitsuba-renderer/mitsuba3/discussions/873

However, for lights specifically, you might be better off (if possible), loading them all into a single scene and then turning them on/off by changing their radiance parameter.

**Reply by foshyjoshy** [reply]
Thank you. This answers my question perfectly.
